iquip/apps/builder.py

```diff
@@ -365,22 +355,146 @@ def get_state(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
                     state[ty] = default
         return state
 
-    def get_procdesc(self, argInfo: Dict[str, Any]) -> Dict[str, Any]:
-        """Gets a procdesc dictionary that describes common parameters of the scannable object.
-
-        Args:
-            argInfo: See argInfo in __init__().
-        """
+    def get_procdesc(self) -> Dict[str, Any]:
+        """Gets a procdesc dictionary that describes common parameters of the scannable object."""
         procdesc = {
-            "unit": argInfo["unit"],
-            "scale": argInfo["scale"],
-            "global_step": argInfo["global_step"],
-            "global_min": argInfo["global_min"],
-            "global_max": argInfo["global_max"],
-            "ndecimals": argInfo["ndecimals"]
+            "unit": self.argInfo["unit"],
+            "scale": self.argInfo["scale"],
+            "global_step": self.argInfo["global_step"],
+            "global_min": self.argInfo["global_min"],
+            "global_max": self.argInfo["global_max"],
+            "ndecimals": self.argInfo["ndecimals"]
         }
         return procdesc
 
+    def value(self) -> Dict[str, Any]:
+        """Overridden.
+        
+        Returns the dictionary of a selected scannable arguments.
+        """
+        selected = self.state["selected"]
+        return self.state[selected]
+
+
+class _RangeScan(QWidget):
+    """A RangeScan widget in Scannable Entry.
+
+    Attributes: See the docstring of Args at __init__ for more detail.
+        scale: See scale in procdesc dictionary at Args.
+        state: See the state docstring at Args.
+        start: QDoubleSpinBox for start arguments inside state.
+        stop: QDoubleSpinBox for stop arguments inside state.
+        npoints: QSpinBox for npoints arguments inside state.
+        randomize: QCheckBox for randomize arguments inside state.
+        layout: The layout of _RangeScan widget.
+    """
+    def __init__(
+        self,
+        procdesc: Dict[str, Any],
+        state: Dict[str, Any],
+        parent: Optional[QWidget] = None
+        ): # pylint: disable=too-many-statements
```

Comment:
This is just question! After you shortened the length of this method, is this Pylint comment necessary?